rfcs/components/moving-from-focus-zone-to-tabster.md

```diff
@@ -0,0 +1,130 @@
+- `className?: string`
+  - _Description:_ Additional class name to provide on the root element, in addition to the ms-FocusZone class.
+  - _Why is no equivalent needed?_ `Tabster` is not a component but a set of utilities, so it has no element to apply a class to.
+- `shouldFocusOnMount?: boolean`
+  - _Description:_ Determines if a default tabbable element should be force-focused on `FocusZone` mount.
+  - _Why is no equivalent needed?_ There is no "mounting" process anymore since `Tabster` is not a React component.
+- `disabled?: boolean`
+  - _Description:_ If set, the
+  - _Why is no equivalent needed?_ There is no "mounting" process anymore since `Tabster` is not a React component.
+- `as?: React.ElementType`
+  - _Description:_ A component that should be used as the root element of the `FocusZone` component.
+  - _Why is no equivalent needed?_ There is no component to replace the root element of via `as`.
+- `shouldEnterInnerZone?: (ev: React.KeyboardEvent<HTMLElement>) => boolean`
+  - _Description:_ Callback function that will be executed on keypresses to determine if the user intends to navigate into the inner (nested) zone. Returning true will ask the first inner zone to set focus.
+  - _Why is no equivalent needed?_ Since `Tabster` is not a component but a set of utilities there is no concept of "nesting".
+- `allowFocusRoot?: boolean`
+  - _Description:_ Allows focus to park on root when focus is in the `FocusZone` at render time.
+  - _Why is no equivalent needed?_ There is no component with a root to park at.
+
+#### Props that may need an equivalent
+
+##### IFocusZone
+
+- `focusElement(childElement?: HTMLElement, forceAlignment?: boolean): boolean`
+  - _Description:_ Sets focus to a specific child element (`childElement`) within the zone. This can be used in conjunction with `shouldReceiveFocus` to create delayed focus scenarios (like animate the scroll position to the correct location and then focus). If `forceAlignment` is true, then focus alignment will be set according to the element provided. Returns true if focus could be set to an active element and false otherwise.
+  - _What should we do about this prop?_ There is probably not a need to have something that matches this functionality 1:1. However, forcing alignment and the ability to specify a child element to get focus are probably things that we want to have something to transition to.
+- `setFocusAlignment(point: Point): void`
+  - _Description:_ Forces horizontal alignment in the context of vertical arrowing to use a specific point (`point`) as the reference, rather than a center based on the last horizontal motion.
+  - _What should we do about this prop?_ We may need something similar to force horizontal alignment, although I am not so sure about using a specific point as reference.
+
+##### IFocusZoneProps
+
+- `defaultTabbableElement?: string | ((root: HTMLElement) => HTMLElement)`
```

Comment:
I definitely don't think we should port this behaviour in the current form , relying on `Tabster` focus finders guarantee some kind of safety/consistency in finding focusables.

Any support IMO would come from our end with `react-tabster` rather than `Tabster` itself, but it is too early to make a definitive call on that

Reply:
Yeah, I never thought it would be ported in the current form, it was more about "we might need an alternative take on this".

Agreed that it probably comes from `react-tabster` rather than `Tabster` too.

Reply:
I have added a note about how it is to early to make a decision here but even if done the API would have to look very different.